### ISSUE TITLE
Android: Sits on "Connecting to Vysor on Android..." screen and does not connect when device is in Portrait mode

### ISSUE THREAD

**Issue by lleworc** [koush/vysor.io#1270]
Vysor Bug Report Link:
https://gist.github.com/VysorLogs/385391019c5bd2b53afb8898855e62e1.js

To Reproduce:
1. Connect mediaDROID+ Android box to computer using USB cable (device is set to portrait mode and PTP USB mode)
2. Open Vysor and select View Device
3. A new window opens and it sits on "Connecting to Vysor on Android..." It hangs on this screen and never loads. It also seems to completely freeze up the Android unit until I exit the attempted connection
4. If I manually change the display orientation to landscape mode Vysor connects fine but I need it to work in portrait mode as well

Expected behavior:
Expect the screen to load in portrait mode with full remote control just as it does in landscape mode

Screenshots:
![Screenshot 2023-06-03 222254](https://github.com/koush/vysor.io/assets/2946727/d8d63e43-294e-4db3-81ac-d503a58db77e)
Landscape Mode (Working):
![Screenshot 2023-06-03 222459](https://github.com/koush/vysor.io/assets/2946727/f13114b6-77cc-4c8a-b37d-3ac9bb50495a)
Portrait Mode (Not Working):
![Screenshot 2023-06-03 222355](https://github.com/koush/vysor.io/assets/2946727/29514a94-f37a-44c8-9566-89072fc637e4)

Desktop:
 - OS: Windows 11
 - Vysor Pro 5.0.16 (Runtime 5.0.7)

Android:
 - Device: mediaDROID+ Android box
 - OS: Android 9
 - Vysor Bug Report Link: https://gist.github.com/VysorLogs/385391019c5bd2b53afb8898855e62e1.js

Reference to similar issue:
#1093 Android: Hangs on "Connecting to Vysor on Android..." screen if initially in Portrait mode.

**Comment by lleworc** [comment]
#1093

**Comment by koush** [comment]
Try lowering the resolution to 25% and test

**Comment by lleworc** [comment]
I tried lowering the resolution from 1080P as recommended but it still does not work while in portrait mode.

**Comment by lleworc** [comment]
I realize now that you are referring to the resolution setting within the Vysor settings. It was set for High Quality settings (as Vysor had suggested the first time I connected to it) with a 75% resolution. As soon as I dropped it down to 50% resolution it started to work in portrait mode. Is there any way to fix this going forward so that it will work in the suggested High Quality mode as well? That mode always worked on the older Android boxes I used before these units. Thank you for your time and assistance it is greatly appreciated.
